cmake/cpp_bindgen.cmake.in

```diff
@@ -30,6 +30,8 @@ include_guard(GLOBAL)
 
 option(GT_ENABLE_BINDINGS_GENERATION "If turned off, bindings will not be generated." ON)
 
+if( NOT TARGET cpp_bindgen_interface )
```

Comment:
Guard that prevents these libraries being added twice when GridTools is in the same overal cmake project as downstream projects that do a find_package( GridTools ), as GridToolsConfig.cmake includes this file.

Reply:
Do you understand why the include_guard above doesn’t do its job in your setup?

Reply:
Indeed I don't understand why that didn't work...
I had errors that stated that I could not add library ```cpp_bindgen_interface``` as it was already defined.
But I guess regardless this file should be included as part of ```find_package( GridTools )``` as it defines functions further down that otherwise would not be exported I guess.